LICENSE.md

```diff
@@ -0,0 +1,25 @@
+# License
+
+As a work of the [United States government](https://www.usa.gov/), this project is in the public domain within the United States of America.
+
+Additionally, we waive copyright and related rights in the work worldwide through the [CC0 1.0 Universal public domain dedication](https://creativecommons.org/publicdomain/zero/1.0/).
```

Comment:
@jayaneti1 I wasn't certain about this, but it seems to be on all the 18F repos. Does this worldwide waiver apply?

Reply:
Good question.  Otherwise looks good.

Reply:
I believe a worldwide waiver is appropriate